test/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ function(tbb_add_test)
     # Define the target for test
     add_executable(${_tbb_test_TARGET_NAME} ${_tbb_test_SUBDIR}/${_tbb_test_NAME}.cpp)
     target_include_directories(${_tbb_test_TARGET_NAME} PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/.. ${CMAKE_CURRENT_SOURCE_DIR})
+    set_property(TARGET ${_tbb_test_TARGET_NAME} PROPERTY ENABLE_EXPORTS TRUE)
```

Comment:
Is this change on purpose? I don't see any connection to bumping the CMake version. If such a connection exist, would be good to add it to the commit message.

Reply:
done